### DISCUSSION TITLE
Cannot get my application reviewed.

### DISCUSSION THREAD

**Discussion by yanmarvin** [community/community#42364] [GitHub Education]
Hi Github,

I am a faculty in a University. The pre-check function does not work. I keep receiving a warning saying that my information shall be added to the bio. However, I have uploaded an official document with the stamp and required information on it, but the OCR seems to be confused by it. Probably due to the small font size and a large paragraph of text. My staff card does not have the enrollment date on it. Can you provide an alternative channel that can check the document by humans instead of the algorithm? Thanks a lot.

**Comment by SilasRodrigues19** [comment]
Hi @yanmarvin 

Try taking a picture instead of uploading, when I tried it was the only way it worked.

**Reply by yanmarvin** [reply]
Dear Silas,

Thanks for your reply. I can only take a photo on the verification webpage. I have tried all different devices, including iPhone, Pixel phone, Macbook, and a Logi webcam. They all do not work. I think it is because the document I have is a letterhead paper with a large chunk of text to include all the information required by Github Education. I really hope Github can provide an alternative way for me to submit the document instead of passing this unreliable OCR as the preliminary check. Thanks.

**Comment by hiten36** [comment]
Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field.

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159
